### PR TITLE
Add project

### PR DESCRIPTION
**Pull request by ali-7**
relates to #10

### CODE REVIEW

File: public/js/projects.js [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+const descriptionError = document.getElementById('descriptionError');
+const gitError = document.getElementById('gitError');
+const webError = document.getElementById('webError');
+const imgError = document.getElementById('imgError');
+const addProjectButton = document.getElementById('addproj');
+const URL = window.location.href;
+const splitUrl = URL.split('/');
+const cohortId = splitUrl[splitUrl.length - 2];
 
 const students = () => {
```
Comment (NouraldinS) [inline public/js/projects.js:19]: You have repeated functions in other files, like your displayErr function, checkName, check* ... etc with your `cohort.js`. Do not repeat functions, reuse.

File: src/database/queries/addProject.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const dbConnection = require('../db_connection');
+
+const addProjectQuery = (newProject, cb) => {
+  const sql = {
+    text: 'INSERT INTO project (name, description, githublink, websitelink, img_url, cohort_id) values ($1,$2,$3,$4,$5,$6) RETURNING name',
+    values: [newProject.nameVal, newProject.descriptionVal, newProject.gitLinkVal,
```
Comment (NouraldinS) [inline src/database/queries/addProject.js]: Fix indentation.

**Review by NouraldinS — CHANGES_REQUESTED** [review]

File: src/database/queries/addProject.js (hunk 2 above)
Comment (ali-7) [reply src/database/queries/addProject.js]: there's no indentation issue

Comment (NouraldinS) [reply src/database/queries/addProject.js]: There is, you cannot add newlines after certain commas in some array\object and leave the rest, it is either you add new lines after commas, or you don't. @ali-7

File: public/js/projects.js (hunk 1 above)
Comment (ali-7) [reply public/js/projects.js:19]: i add the same function i used in one file

**Review by NouraldinS — APPROVED** [review]
@ali-7 You can merge this one.

**Pull request merged**